experiments/les_strats/config.jl

```diff
@@ -0,0 +1,136 @@
+"""Custom calibration configuration file."""
```

Comment:
```suggestion
#= Custom calibration configuration file. =#
```
Doc strings need to be attached to something, this should probably just be a comment